### PR TITLE
Removed cfengine determine-version which is no longer needed

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @vpodzime can review this?

File: travis-scripts/script.sh [hunk 1]
```diff
@@ -4,14 +4,6 @@ set -x
...
 cd $TRAVIS_BUILD_DIR || exit 1
 
-# Unshallow the clone. Fetch the tags from upstream even if we are on a
-# foreign clone. Needed for determine-version.py to work, specifically
-# `git describe --tags HEAD` was failing once the last tagged commit
-# became too old.
-git fetch --unshallow
-git remote add upstream https://github.com/cfengine/core.git  \
-    && git fetch upstream 'refs/tags/*:refs/tags/*'
-
```
Comment (vpodzime) [inline travis-scripts/script.sh]: Trying to sneak things in? :wink:

Comment (olehermanse) [reply travis-scripts/script.sh]: No, it was there because determine-version needed it.

Comment (olehermanse) [reply travis-scripts/script.sh]: It even says so in the comment.

Comment (vpodzime) [reply travis-scripts/script.sh]: OH, wow, that didn't even cross my mind!

**Pull request merged**